python/cudf/dataframe/series.py

```diff
@@ -543,7 +543,7 @@ def astype(self, dtype):
 
         return self._copy_construct(data=self._column.astype(dtype))
 
-    def argsort(self, ascending=True):
+    def argsort(self, ascending=True, na_position="last"):
```

Comment:
Haven’t really looked into the new libucdf code beyond the api, is it still parallel radixsort? I ask since the description comment still mentions it here

Reply:
I would be surprised if it is not radix sort for primitive types, since it is way more efficient than a comparison sort when you don't need a comparison sort...

Reply:
I'm not sure what's underneath the Thrust sort, but we shouldn't speculate. Updated the docstring to indicate it uses Thrust sort.